FrontEnd/src/components/profileList/ProfileCard.jsx

```diff
@@ -89,12 +112,18 @@ export default function ProfileCard({ isAuthorized, data }) {
           </div>
         </div>
       </Link>
-      <StarForLike
-        isSaved={isSaved}
-        isAuthorized={isAuthorized}
-        ownProfile={ownProfile}
-        handleClick={isSaved ? handleDeleteSaved : handleSave}
-      ></StarForLike>
+      <div className={css['bell-container']}>
+        <BellForUpdates
+          className={savedIsUpdated ? '' : 'hidden'}
```

Comment:
Do we need it? We do not use `className` prop in the `BellForUpdates`

Reply:
Deleted redundant code